src/main/java/de/hysky/skyblocker/config/SkyblockerConfig.java

```diff
@@ -873,23 +876,65 @@ public static class DwarvenMines {
 
 		@SerialEntry
 		public DwarvenHud dwarvenHud = new DwarvenHud();
+
+		@SerialEntry
+		public CrystalsHud crystalsHud = new CrystalsHud();
+
+		@SerialEntry
+		public CrystalsWaypoints crystalsWaypoints = new CrystalsWaypoints();
 	}
 
 	public static class DwarvenHud {
 		@SerialEntry
-		public boolean enabled = true;
+		public boolean enabledCommissions = true;
+
+		@SerialEntry
+		public boolean enabledPowder = true;
 
 		@SerialEntry
 		public DwarvenHudStyle style = DwarvenHudStyle.SIMPLE;
 
 		@SerialEntry
 		public boolean enableBackground = true;
 
+		@SerialEntry
+		public int commissionsX = 10;
+
+		@SerialEntry
+		public int commissionsY = 10;
+
+		@SerialEntry
+		public int powderX = 10;
+
+		@SerialEntry
+		public int powderY = 70;
+	}
+	public static class CrystalsHud {
+		@SerialEntry
+		public boolean enabled = true;
+
+		@SerialEntry
+		public boolean showLocations = true;
+
+		@SerialEntry
+		public int locationSize = 8;
+
 		@SerialEntry
 		public int x = 10;
 
 		@SerialEntry
-		public int y = 10;
+		public int y = 130;
+	}
+	public static class CrystalsWaypoints {
+		@SerialEntry
+		public boolean enabled = true;
+
+		@SerialEntry
+		public boolean findInChat = true;
+		@SerialEntry
+		public Map<String, CrystalsWaypoint> ActiveWaypoints = new HashMap<>() {};
```

Comment:
This shouldn't be in the config - move it to one of the classes because the locations can change and shouldn't be serialized into the config file.